packages/terra-form-input/src/InputField.jsx

```diff
@@ -180,6 +187,7 @@ const InputField = (props) => {
         id={inputId}
         type={inputType}
         onChange={onChange}
+        placeholder={inputAttrs.placeholder || placeholder}
```

Comment:
I would expect placeholder to be valued first if both were provided since it's a first class prop

Reply:
For passivity the inputAttrs are valued first. The order probably isn't as important because it is only an issue if both attributes are provided which consumers can easily reconcile, but I think it makes sense to prioritize the existing API when a change like this is made that can override the old API. This is how disabled is being used now in all of the form fields, for better or for worse.

Reply:
I went ahead and flipped these: https://github.com/cerner/terra-core/pull/2676/commits/a9c6e70ac392f5a2e6019746106660d581c435a9